smack-core/src/main/java/org/jivesoftware/smack/ReconnectionManager.java

```diff
@@ -87,15 +88,60 @@ public static void setEnabledPerDefault(boolean enabled) {
      *
      * @return true if new connection will come with an enabled reconnection mechanism
      */
-    public static boolean getEnabledPerDefault() {
-        return enabledPerDefault;
+    public static boolean getEnabledByDefault() {
+        return enabledByDefault;
     }
 
     // Holds the connection to the server
     private final WeakReference<AbstractXMPPConnection> weakRefConnection;
     private final int randomBase = new Random().nextInt(13) + 2; // between 2 and 15 seconds
     private final Runnable reconnectionRunnable;
-
+    
+    private static int defaultFixedDelay = 15;
+    private static ReconnectionPolicy defaultReconnectionPolicy = ReconnectionPolicy.RANDOM_INCREASING_DELAY;
+    
+    private volatile int fixedDelay = defaultFixedDelay;
+    private volatile ReconnectionPolicy reconnectionPolicy = defaultReconnectionPolicy;
+    
+    
+    /**
+     * Set the default fixed delay in seconds between the reconnection attempts. 
+     * Also set the default connection policy to FIXED_DELAY
```

Comment:
Should use {@link ReconnectionPolicy#FIXED_DELAY} (not sure if the syntax is correct but you get the idea)